spec/factories/external_users.rb

```diff
@@ -17,11 +17,12 @@
   factory :external_user do
     transient do
       build_user { true }
+      last_name { Faker::Name.last_name }
     end
 
     after(:build) do |eu, evaluator|
       if evaluator.build_user
-        eu.user ||= build(:user, first_name: Faker::Name.first_name, last_name: Faker::Name.last_name, email: Faker::Internet.email, password: 'password', password_confirmation: 'password')
+        eu.user ||= build(:user, first_name: Faker::Name.first_name, last_name: evaluator.last_name, email: Faker::Internet.email, password: 'password', password_confirmation: 'password')
```

Comment:
will this not need to fallback if the evaluator has not supplied a last name
```suggestion
        eu.user ||= build(:user, first_name: Faker::Name.first_name, last_name: evaluator.last_name || Faker::Name.last_name, email: Faker::Internet.email, password: 'password', password_confirmation: 'password')
```

Reply:
If this is not set in the `create` or `build` then it gets set on line 20, above.

Reply:
right! i'd be tempted to go further then and make the first_name, and even email transient with default, as for last name. would be more consistent and less confusing.

Reply:
I could do this. I was going for the minimum possible change but it makes sense to be able to set the other fields too.

Reply:
Actually, I had second thought and made it simpler with bb6347a86908a8e1c86d868bca1065203f7228a5